src/components/organizations/index.jsx

```diff
@@ -1,26 +1,81 @@
-import {useEffect, useState } from "react";
-import { getOrganizations } from "../../api/endpoints";
+import { useEffect, useState } from "react";
 import OrganizationsList from "../common/OrganizationsList";
-import {Box, Typography, CircularProgress} from "@mui/material";
+import { Box, Typography, CircularProgress, Stack, Button, Link } from "@mui/material";
+import BasicDialog from "../common/BasicDialog";
+import GroupAddOutlinedIcon from "@mui/icons-material/GroupAddOutlined";
+import { createNewOrganization, getOrganizations } from "../../api/endpoints/apiService";
 
 import { vars } from "../../theme/variables";
-const { gray700 } = vars;
+const { gray600, gray700, brand700, brand800 } = vars;
+
+const linkStyles = {
+  color: brand700,
+  fontWeight: 600, 
+  textDecoration: "none", 
+  "&:hover": { 
+    color: brand800 
+  }
+}
 
 const Organizations = () => {
   const [organizations, setOrganizations] = useState([]);
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState(false);
+  const [open, setOpen] = useState(false);
+  const [message, setMessage] = useState();
+
+  // TODO: change this to be dynamic when we get the response from api call
+  const groupname = "aigul"
 
   const fetchOrganizations = async() => {
```

Comment:
@Aiga115 this code is exactly the same we have in Dashboard/Prganizations/index.jsx, could you factor this out in a util module so that we can re-use the same code here and in the other file? If you need to you can pass the hooks as parameter, but at least we avoid to repeat code that needs to be maintained. Thanks